packages/blocks/src/class-blocks.php

```diff
@@ -11,32 +11,42 @@
 
 namespace Automattic\Jetpack;
 
+use Jetpack_Gutenberg;
+
 /**
  * Register and manage blocks within a plugin. Used to manage block registration, enqueues, and more.
  *
  * @since 9.0.0
  */
 class Blocks {
 	/**
-	 * Wrapper function to safely register a gutenberg block type
-	 *
-	 * @param string $slug Slug of the block.
-	 * @param array  $args Arguments that are passed into register_block_type.
+	 * Wrapper function to safely register a Gutenberg block type
 	 *
 	 * @see register_block_type
 	 *
-	 * @since 6.7.0
+	 * @since 9.0.0
+	 *
+	 * @param string $slug Slug of the block.
+	 * @param array  $args {
+	 *     Arguments that are passed into register_block_type.
+	 *     See register_block_type for full list of arguments.
+	 *     Can also include 2 extra arguments not currently supported by register_block_type.
+	 *
+	 *     @type array $version_requirements Array containing required Gutenberg version and, if known, the WordPress version that was released with this minimum version.
```

Comment:
Should this refer to `is_gutenberg_version_available` for the format of the array? The WordPress doc standards seem to say that.

Reply:
👍 07cf13afef3cd00b9d2e37d6bc39cdae135441a3